kitsune/products/migrations/0001_squashed_0020_remove_sumoplaceholderpage_page_ptr_and_more.py

```diff
@@ -197,6 +207,7 @@
                     "focus-firefox",
                     "thunderbird",
                     "thunderbird-android",
+                    "firefox-enterprise",
```

Comment:
`Relay` also needs to be added to the `Crashing and slow performance` topic product list.